### DISCUSSION TITLE
(tvOS) How to focus on elements behind an absolute View

### DISCUSSION THREAD

**Discussion by benoitzohar** [react-native-tvos/react-native-tvos#483] [Q&A]
Hi,

I'm trying to figure out how I could focus on elements that are behind a View with `position: 'absolute'`.

I'm not sure this is a bug or not, but when you add an absolute `<View />` on top of a `<TouchableOpacity />`, the element doesn't get focused.

Note that I've tried to use `isTVSelectable={false}` on the absolute `<View />`, without any luck.

Here's an example of what I mean: I have a bunch of `<TouchableOpacity />` in a calendar and if I add an empty absolute view on top of it (with `StyleSheet.absoluteFillObject`) the cells are not focusable anymore.

| Without Absolute View	| With Absolute View 	|
|---	|---	|
|   <video src="https://user-images.githubusercontent.com/1698042/228953469-91054e41-60c4-4464-8906-8529b6f190f4.mp4" /> |  <video src="https://user-images.githubusercontent.com/1698042/228953548-b5b7ec24-2877-44b4-874d-8820cb88762b.mp4" />	| 

For context, I **have** to use an `absolute` layout to display the events properly in the grid. Eventually I want to display the events on top of the days, but still be able to focus on the days.

Alternatively, the focus on `<TouchableOpacity />` with `position: absolute` doesn't work at all, as reported [in this issue](https://github.com/react-native-tvos/react-native-tvos/issues/71). 

Here's a code to reproduce:
```tsx
const ADD_ABSOLUTE_VIEW = true;

export function Repro() {
  return (
    <View
      style={{alignItems: 'center', justifyContent: 'center', height: '100%'}}>
      <TouchableOpacity
        style={{height: 300, width: 300, backgroundColor: 'red'}}
        onFocus={() => console.log('onFocus')}>
        <Text>Focus me</Text>
      </TouchableOpacity>
      {ADD_ABSOLUTE_VIEW && <View style={StyleSheet.absoluteFillObject} />}
    </View>
  );
}

```

**Comment by qwerty2k** [comment]
Just to add to this, i wanted to have a 'floating' side menu that could overlap content and then pressing off it should shrink the menu down, but im also having problems when trying to navitgate to the items once a position absolute object is rendered.

**Reply by kim1350** [reply]
Found a solution?
